src/types/options/qualityRequestOptions.ts

```diff
@@ -222,7 +224,9 @@ export interface MathCriterion extends BaseCriterion {
 interface BaseEdgeStatisticsCriterion extends BaseCriterion {
   type: 'edgeStatistics'
   edgeStatisticsServiceUrl?: string
-  edgeStatisticGroupId: number
+  /** @deprecated use edgeStatisticCollectionId id instead */
+  edgeStatisticGroupId?: number
```

Comment:
can you already remove this `edgeStatisticGroupId` from your library? so we can remove it soon from the service

Reply:
done